### ISSUE TITLE
Not working for resource (room) calendars

### ISSUE THREAD

**Issue by reinhrst** [microsoftgraph/microsoft-graph-docs#3997]
When trying calendarView/delta for a personal calendar, everything works fine, including getting updates. When trying it on a room calendar that the user is delegated to, it gets the first page fine, but the "nextLink"-page returns an "Access is denied"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: da37d83f-dd2b-dd85-eb5d-75208e26412d
* Version Independent ID: aa5f5279-bec8-fe6e-ab1b-8dc87f2ed688
* Content: [Get incremental changes to events in a calendar view  - Microsoft Graph](https://docs.microsoft.com/en-us/graph/delta-query-events)
* Content Source: [concepts/delta-query-events.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/concepts/delta-query-events.md)
* Product: **non-product-specific**
* GitHub Login: @piotrci
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by nicelord** [comment]
yes.. I am experiencing the same problem here using delegated access.
even with global administrator and adjust app permissions to higher access

**Comment by mcolmant** [comment]
Same problem here. Any updates on this???

**Comment by natsning** [comment]
Same here. Please solve this soon.

**Comment by EsaHeikkiPoutanen** [comment]
sample second request doesn't compile in c#, .SkipToken is not recognized...

**Comment by cbass27** [comment]
Any update on this? Problem still exists.
